framework-plugins/lisk-framework-dashboard-plugin/src/ui/utils/index.ts

```diff
@@ -60,12 +70,14 @@ export const updateStatesOnNewTransaction = (
 	return [transaction, ...unconfirmedTransactions].slice(-1 * MAX_TRANSACTIONS) as Transaction[];
 };
 
-export const getApplicationUrl = async () => {
+export const getConfig = async (prop: keyof Config) => {
```

Comment:
instead of returning per property, i think it's better to return complete config to avoid call

Reply:
Application name and URL are fetched once so 2 calls in total. I don't feel that remedying 2 calls' overhead is worth complicating MainPage's complexity and decreasing it's clarity by introducing usage of joint config calls. Are you sure of it? Or do you have something in your mind that is a very clear solution?

Reply:
If we increase the config properties, it will linearly increase the call.
Also, calling config endpoint once in the root as it was done originally does not seem to be increase of complexity